.github/workflows/e2e_lighthouse_performance_acceptance_tests.yml

```diff
@@ -319,8 +319,15 @@ jobs:
             - logged-out-user/click-all-buttons-on-about-foundation-page
             - logged-out-user/click-all-buttons-on-donation-thanks-page
             - logged-out-user/click-all-buttons-on-navbar
+            - logged-out-user/click-all-buttons-on-partnerships-page
+            - logged-out-user/click-all-buttons-on-teach-page
+            - logged-out-user/click-all-buttons-on-volunteer-page
             - logged-out-user/click-all-links-in-about-oppia-footer
             - logged-out-user/click-all-links-on-get-started-page
+            - logged-out-user/check-all-user-flow-of-donor
+            - logged-out-user/check-all-user-flow-of-parent-teacher
+            - logged-out-user/check-all-user-flow-of-partner
+            - logged-out-user/check-all-user-flow-of-volunteer
```

Comment:
Given that these tests are written properly as user journeys, shouldn't they be consistent with names of other tests i.e. shouldn't it encapsulate the crux of the actions performed by the user?
@imchristie

Reply:
Leaving this to @imchristie, since she has suggested the names.

Reply:
I suggested using the word "user flow" because Akash used it in his proposal. So it looks good to me. 
cc/ @Lawful2002